### ISSUE TITLE
[Bug?] By buying a item on darkweb you can get duplicate programs

### ISSUE THREAD

**Issue by Guillermo-Santos** [danielyxie/bitburner#2798]
When i let making relaySMTP.exe i saw that i had produced enought money to buy it, so i went and did it, but i saw that the creation process was not canceled and did it myself. 

By using de command ls to look for a file name i saw that there was a duplicate of the program.

- [ ] Steps to reproduce
> - Start program creation process
> - Press "Do something else simultaneously"
> - Go to terminal
> - Connect darkweb
> - Buy program
> - Stop creation process or let it finish
> - Once it is done "ls" on home to see files

- [ ] Version of the game, e.g. Bitburner v1.4.0 (49de4a28)
- [ ] Captures
![bug](https://user-images.githubusercontent.com/96314559/150889342-010b39e1-cb42-4154-b92f-2f8766ccfa62.jpg)

**Comment by phyzical** [comment]
haha, well.. you can also rm programs if you want. i would argue it really doesnt have that much of an effect if you end up with two of the same program.

ill leave this one to higher powers to decide
